corehq/apps/app_manager/static/app_manager/js/widgets_v4.js

```diff
@@ -28,7 +28,7 @@ hqDefine("app_manager/js/widgets_v4", [
                     return {
                         results: _.map(data.apps, function (build) {
                             return {
-                                id: build.id,
+                                id: build.version,
```

Comment:
This breaks the build comparison on the view source files page, which is still expecting a build id.

Reply:
added, https://github.com/dimagi/commcare-hq/pull/23771